.github/workflows/build.yml

```
@@ -185,3 +185,62 @@ jobs:
     - run: make clang-debug
     - name: tests (wasm2c tests excluding memory64)
       run: ./test/run-tests.py wasm2c --exclude-dir memory64
+
+  build-arch:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        arch: [s390x]
+    services: # still faster on debian...
+      distcc:
+        image: debian:latest
+        options: --health-cmd distccmon-text --health-interval 5s --health-start-period 5m debian:latest bash -c "apt-get update && apt-get install -y g++-s390x-linux-gnu distcc && distccd --daemon --no-detach"
+        ports:
+          - 3632:3632
+    steps:
```

Another simplification:

```suggestion
    env:
      CMAKE_GENERATOR: Ninja
      QEMU_LD_PREFIX: /usr/${{matrix.arch}}-linux-gnu/
    steps:
```

Then get rid of `-G Ninja` and the other `QEMU_LD_PREFIX` settings below.

---

We use `-G Ninja` elsewhere in this file though... so maybe leave that one as is for consistency?

---

@sbc100 - totally non-critical, just a suggestion!  I think `QEMU_LD_PREFIX` does lower the risk for forgetting it if the workflow has to change.

---

that works...? well, let's find out! (github actions documentation is not exactly the easiest thing to read...)